src/common/base/tests/comparable-test.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import {assert} from '../../../platform/chai-web.js';
+import {compareNulls} from '../comparable.js';
+
+describe('comparable', () => {
+  it('implements compareNulls', () => {
```

Comment:
It would be worthwhile to add tests for comparing `undefined`s.

Reply:
done